site/pages/api/duplicate-disruption.api.ts

```diff
@@ -105,7 +106,19 @@ const duplicateDisruption = async (req: NextApiRequest, res: NextApiResponse): P
         if (draftDisruption.socialMediaPosts) {
             await Promise.all(
                 draftDisruption.socialMediaPosts.map(async (socialMediaPost) => {
-                    await upsertSocialMediaPost(socialMediaPost, session.orgId, session.isOrgStaff);
+                    await upsertSocialMediaPost(
+                        {
+                            ...socialMediaPost,
+                            ...(socialMediaPost.accountType === "Hootsuite"
+                                ? { publishDate: socialMediaPost.publishDate || defaultDateTime().date }
```

Comment:
Dont see an issue with this but when you add a new social medial post to a disruption without date and time, publish it and then click duplicate disruption - the new disruption has social media post with default time set to 10min ahead. Previously this was not the case.

Should the 10min ahead be enabled only for disruptions created from a template?

Reply:
yes this is expected behaviour